pkg/client/didexchange/client.go

```diff
@@ -191,86 +188,16 @@ func (c *Client) startServiceEventListener() {
 		select {
 		case e := <-c.actionCh:
 			// assigned to var as lint fails with : Using a reference for the variable on range scope (scopelint)
-			msg := e
-			c.handleActionEvent(&msg)
+			c.GetActionEvent() <- e
```

Comment:
This comment should have been in the prior PR:

Simple accessors should not have a Get prefix - they are not Getting anything.

Reply:
I am going to change it! Do not merge!

Reply:
Done!